CONTRIBUTING.md

```diff
@@ -2,12 +2,15 @@
 
 🎉 Thanks for considering contributing to this project! 🎉
 
-These guidelines will help you send a pull request.
+When contributing to this repository, please first discuss the change you wish to make via an
+[issue](https://github.com/netlify/next-runtime/issues/new/choose). Please use the issue templates. They are there to
+help you and to help the maintainers gather information.
 
-If you're submitting an issue instead, please skip this document.
+Before working on an issue, ask to be assigned to it. This makes it clear to other potential contributors that someone
+is working on the issue.
 
-If your pull request is related to a typo or the documentation being unclear, please click on the relevant page's `Edit`
-button (pencil icon) and directly suggest a correction instead.
+When creating a PR, please fill out the template to the best of your abilities. Do not delete it. The information in the
+template helps maintainers review your pull request.
```

Comment:
[sand] not a big deal, but maybe this sounds a bit heavy handed?
```suggestion
When creating a PR, please use the template. The information in the template helps maintainers review your pull request.```

Reply:
+1